relation_engine_server/test/test_spec_loader.py

```diff
@@ -7,20 +7,37 @@
 from relation_engine_server.utils import spec_loader
 from relation_engine_server.utils.spec_loader import SchemaNonexistent
 from relation_engine_server.utils.config import get_config
-from relation_engine_server.utils.wait_for import wait_for_api
-
-_CONF = get_config()
-_TEST_DIR = os_path.join('/app', 'relation_engine_server', 'test', 'data')
 
 
 class TestSpecLoader(unittest.TestCase):
 
     @classmethod
     def setUpClass(cls):
-        wait_for_api()
-        cls.config = get_config()
+        cls.test_dir = os_path.join('/app', 'relation_engine_server', 'test')
+        cls.test_spec_dir = os_path.join(cls.test_dir, 'spec_release', 'sample_spec_release')
+
+        config = get_config()
+        cls.repo_path = config['spec_paths']['repo']
+        for key in config['spec_paths'].keys():
+            if cls.repo_path in config['spec_paths'][key]:
+                config['spec_paths'][key] = config['spec_paths'][key].replace(
+                    cls.repo_path,
+                    cls.test_spec_dir
+                )
+        cls.config = config
```

Comment:
Edit `config` settings to point to the `sample_spec_release` dir for the duration of this test.